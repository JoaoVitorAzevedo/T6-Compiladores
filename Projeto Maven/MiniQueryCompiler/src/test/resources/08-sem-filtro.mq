# O gerador de JS atual vai tentar usar 'filteredData', que não foi definida.
# O teste passa porque valida o código gerado, expondo essa falha.
LOAD "data.json"
SELECT .nome, .email