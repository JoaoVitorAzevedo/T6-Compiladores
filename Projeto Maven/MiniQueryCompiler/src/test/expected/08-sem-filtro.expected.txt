✔ ANÁLISE SEMÂNTICA VÁLIDA

CÓDIGO JS GERADO:
// Código gerado automaticamente
const fs = require('fs');

const input = JSON.parse(fs.readFileSync('data.json', 'utf-8'));
const result = input.map(item => ({
  nome: item.nome,
  email: item.email
}));